frontend/package.json

```diff
@@ -31,6 +31,7 @@
     "axios": "^1.4.0",
     "clsx": "^2.1.1",
     "leaflet": "^1.9.4",
+    "leaflet-switch-basemap": "^1.0.6",
```

Comment:
This library is not being maintained for last 2 years, I would recommend we bring the code into our own repo or find another library.

Reply:
Yeah... I was kinda heading there already. What was supposed to save us some effort has turned into a headache.

In researching available plugins I wasn't able to find anything better. Especially in the Leaflet/React ecosystem. Let me know otherwise. 

As for now, I'll bring in what I can and remove the dependency.

Reply:
It's gone now.